packages/mock-block-dock/dev/dev-app.tsx

```diff
@@ -1,6 +1,6 @@
 import { Entity } from "@blockprotocol/graph";
 import { ChangeEvent, FunctionComponent, useState } from "react";
-import { render } from "react-dom";
+import { createRoot } from "react-dom/client";
```

Comment:
Why're we importing from `react-dom/client` and not `react-dom`?

Reply:
That's what the code example I found did, e.g. [here](https://reactjs.org/blog/2022/03/08/react-18-upgrade-guide.html). Happy to change it if it matters

Reply:
Oh I misread something elsewhere. Ignore me!